jsalyer23/oop_pokedex
# Minitest Error

Title: Help with Minitest error

Body:

I keep getting an error when I try to run a test for the `.traits` method in my `Pokemon` class.
1. Go to `tests/unit/hkhu_test.rb`
2. Run `bundle exec rake`

Expected: The Minitest runs as expected with green or red `.` and prints out information about the test.

Actual: I get a yellow "E" for error. This is the error message that's returned:

  1) Error:
PokemonTest#test_traits:
TypeError: class or module required
    /Users/joshsalyer/Code/oop_pokedex/tests/unit/hkhu_test.rb:22:in `test_traits'

I've tried using `binding.pry` to check if my Pokemon object is being created in setup and what the value of `.traits` when it's called on that object. Both have the value i'm expecting but the test still doesn't run as expected.

Here is the [test file](https://github.com/jsalyer23/oop_pokedex/blob/gumby/tests/unit/hkhu_test.rb)
